src/fiuba/algo3/tp2/PicoMetal.java

```diff
@@ -0,0 +1,10 @@
+package fiuba.algo3.tp2;
+
+import fiuba.algo3.tp2.estrategiasDesgaste.EstrategiaDesgasteAbrupta;
+
+public class PicoMetal extends Pico {
+    public PicoMetal() {
+        this.fuerza = 12;
+        this.desgastador = new EstrategiaDesgasteAbrupta(400, 10);
```

Comment:
hay algun motivo por el cual aca le pasas la durabilidad como un literal envez de pasarle this.durabilidad?

Reply:
Si, eran las 4 de la maniana